### PR TITLE
bmp: Stats reports for Adj-RIBs-In and Loc-RIB routes

### PR DESCRIPTION
**Pull request by iwaseyusuke**
This PR enables to send BMP statistics reports for the number of routes in Adj-RIBs-In and Loc-RIB.
Also includes the improvements of BMP statistics reports serializer and the updates of documents for BMP configurations.

### CODE REVIEW

File: server/bmp.go [hunk 1]
```diff
@@ -113,16 +113,27 @@ func (b *bmpClient) loop() {
...
+			} else if b.c.RouteMonitoringPolicy == config.BMP_ROUTE_MONITORING_POLICY_TYPE_LOCAL_RIB {
 				ops = append(ops, WatchBestPath(true))
 			}
 			w := b.s.Watch(ops...)
 			defer w.Stop()
 
+			var tickerCh <-chan time.Time
+			if b.c.StatisticsTimeout <= 0 {
+				log.WithFields(log.Fields{"Topic": "bmp"}).Debug("statistics reports disabled")
+			} else if 15 <= b.c.StatisticsTimeout && b.c.StatisticsTimeout <= 65535 {
```
Comment (ishidawataru) [inline server/bmp.go]: What about do the validation here?

https://github.com/osrg/gobgp/blob/master/config/default.go#L218-L234

File: tools/pyang_plugins/gobgp.yang [hunk 2]
```diff
@@ -549,6 +549,11 @@ module gobgp {
       type bmp-route-monitoring-policy-type;
       default PRE-POLICY;
     }
+    leaf statistics-timeout {
```
Comment (ishidawataru) [inline tools/pyang_plugins/gobgp.yang]: `statistics-interval` is a better name?

File: server/bmp.go (hunk 1 above)
Comment (iwaseyusuke) [inline server/bmp.go]: Thanks!
I will move this validation to `config/default.go`.

File: tools/pyang_plugins/gobgp.yang (hunk 2 above)
Comment (iwaseyusuke) [inline tools/pyang_plugins/gobgp.yang]: It just follows Junos OS configuration name [`statistics-timeout`](https://www.juniper.net/documentation/en_US/junos/topics/reference/configuration-statement/statistics-timeout-edit-routing-options.html).
Also I picked the range for this value from Junos OS.
(Sorry I don't have strong will...)

Comment (ishidawataru) [inline tools/pyang_plugins/gobgp.yang]: ah, I see. then I think we can keep it as is.

**Comment by fujita** [comment]
Thanks!

**Pull request merged**